src/reflect/scala/reflect/internal/Types.scala

```diff
@@ -419,7 +419,7 @@ trait Types
      *  The empty list for all other types */
     def parents: List[Type] = List()
 
-    /** For a class with nonEmpty parents, the first parent.
+    /** For a class with !isEmpty parents, the first parent.
```

Comment:
sed, or subtle snark?

Reply:
It wasn't meant to be subtle, but I tired of converting the other comments. Who knew nonempty was so often commented? The resultant effect is subtler than intended.